<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>FileEdit</class>
 <widget class="QWidget" name="FileEdit">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>371</width>
    <height>523</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Отредактировать</string>
  </property>
  <widget class="QLabel" name="labelOwnerEdit">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>350</y>
     <width>121</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Владелец(ID)</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButtonFileEdit">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>430</y>
     <width>111</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Сохранить</string>
   </property>
  </widget>
  <widget class="QTextBrowser" name="textBrowserFileEdit">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>30</y>
     <width>311</width>
     <height>461</height>
    </rect>
   </property>
   <property name="html">
    <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'MS Shell Dlg 2'; font-size:7.8pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p align=&quot;center&quot; style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; font-size:10pt; text-decoration: underline;&quot;&gt;Редактирование файла&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
  <widget class="QLabel" name="labelLocationEdit">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>280</y>
     <width>101</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Путь</string>
   </property>
  </widget>
  <widget class="QLabel" name="labelNameEdit">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>140</y>
     <width>81</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Имя файла</string>
   </property>
  </widget>
  <widget class="QLabel" name="labelSizeEdit">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>210</y>
     <width>101</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Размер</string>
   </property>
  </widget>
  <widget class="QLabel" name="labelIdFileEdit">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>70</y>
     <width>81</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>ID</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEditLocationEdit">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>310</y>
     <width>221</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEditOwnerEdit">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>380</y>
     <width>221</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEditNameEdit">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>170</y>
     <width>221</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEditSizeEdit">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>240</y>
     <width>221</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEditIdFileEdit">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>100</y>
     <width>221</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <zorder>textBrowserFileEdit</zorder>
  <zorder>labelOwnerEdit</zorder>
  <zorder>pushButtonFileEdit</zorder>
  <zorder>labelLocationEdit</zorder>
  <zorder>labelNameEdit</zorder>
  <zorder>labelSizeEdit</zorder>
  <zorder>labelIdFileEdit</zorder>
  <zorder>lineEditLocationEdit</zorder>
  <zorder>lineEditOwnerEdit</zorder>
  <zorder>lineEditNameEdit</zorder>
  <zorder>lineEditSizeEdit</zorder>
  <zorder>lineEditIdFileEdit</zorder>
 </widget>
 <resources/>
 <connections/>
</ui>
